### PR TITLE
test: Convert sites-and-appilcations Cypress test over to Playwright

### PR DESCRIPTION
**Pull request by jcbcapps**
# SC-1261

## Proposed changes
Converts the `sites-and-applications.cy.js` test into two Playwright tests, `myspace.spec.ts` and `sites-and-apps.spec.ts`

## Reviewer notes
Make sure that all tests are passing on your local machine.

## Setup
Run the Playwright tests as you normally would:
- `yarn services:up -d`
- `yarn e2e:test`

### CODE REVIEW

File: e2e/cypress/e2e/sites-and-applications.cy.js [hunk 1]
```diff
@@ -1,423 +0,0 @@
...
-      'Finance & Travel',
-      'Outprocessing',
-      'Management',
-      'vMPF',
-      'Supervisory',
-      'Education',
-      'Public Social Platforms',
-      'Public Military Websites',
-    ]
-    collections.forEach((collection) => cy.contains(collection))
```
Comment (gidjin) [inline e2e/cypress/e2e/sites-and-applications.cy.js:47]: Question: Is this test carried over to the new files? Though these are entered by users correct? Probably don't need to validate that all user content is available, just things that are part of our software.

File: e2e/playwright/portal-client/tests/sites-and-apps.spec.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    ).toBeHidden()
+  })
+
+  test('can add collections from the Sites & Applications page to My Space', async ({
+    page,
+    loginPage,
+  }) => {
+    // Login and check that user is in My Space
+    await loginPage.login('user1', 'user1pass')
+    await expect(page.locator('text=WELCOME, BERNIE')).toBeVisible()
```
Comment (gidjin) [inline e2e/playwright/portal-client/tests/sites-and-apps.spec.ts:71]: thought: I wonder if this should be come a utility

File: e2e/cypress/e2e/sites-and-applications.cy.js [hunk 3]
```diff
@@ -1,423 +0,0 @@
...
-        // First undo
-        cy.findAllByRole('button', { name: 'Remove this link' }).first().click()
-        cy.contains('Webmail').should('not.exist')
-        cy.contains('Undo remove').click()
-        cy.contains('Webmail')
-
-        // Don't undo
-        cy.findAllByRole('button', { name: 'Remove this link' }).first().click()
-        cy.contains('Webmail').should('not.exist')
-        cy.contains('Undo remove').should('not.exist')
```
Comment (gidjin) [inline e2e/cypress/e2e/sites-and-applications.cy.js:213]: Question: I don't see this case in the new test. Is it possible to check it as well? Maybe just add it as the final conditions in `can remove multiple links at once from an existing collection`?

**Review by gidjin — APPROVED** [review]
Left a couple questions, but looks good to me and passes locally. Though the article.spec.ts seems to like failing on a second run in a row, which is unrelated to your changes here.

File: e2e/playwright/portal-client/tests/sites-and-apps.spec.ts (hunk 2 above)
Comment (jcbcapps) [reply e2e/playwright/portal-client/tests/sites-and-apps.spec.ts:71]: @gidjin  After our discussion yesterday, I was thinking of holding off on creating a utility for this until we consolidate all of our databases into one area. Thoughts?

**Pull request merged**